src/rpc/blockchain.cpp

```diff
@@ -1512,6 +1512,9 @@ UniValue getchaintxstats(const JSONRPCRequest& request)
         }
     }
 
+    if (pindex->nHeight < 2) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Block's height should be 2 or more.");
```

Comment:
I wonder if the error could be avoided by making the algorithm compatible with 1 block chain.